part01/process.php

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 1</title>
+</head>
+<body>
+
+<?php
+    $weight = $_POST['weight'];
+    $height = $_POST['height'];
+
+    //formula for calculating BMI:
+    //BMI = (Weight in Pounds / (Height in inches x Height in inches)) x 703
+    //source: http://www.bmi-calculator.net/bmi-formula.php
+
+    $BMI = ($weight / ($height * $height)) * 703;
+
+    if($BMI < 18){
+        $Label_Value = "Underweight";
+    } else if($BMI > 30){
+        $Label_Value = "Obsese";
+    } else {
+        $Label_Value =  "Normal";
+    }
```

Comment:
good use of an if/else structure! I like that you leave the  middle value to your else - most efficient 👍